### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by AngelGzS**
## This PR contains:

- Fixes issue #23
- Keeps **public/media** folder

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PORT=3000
+URL=http://localhost:3000
+
+VIDEO_TYPES=.ogv|.mp4|.mpg
```
Comment (itsrachelfish) [inline .env]: Media types are hardcoded due to [browser restrictions](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers), they would not change based on the environment. You should remove these definitions.

File: media-player.js [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+let audioTypes = new Set(
+    process.env.AUDIO_TYPES ? 
+    process.env.AUDIO_TYPES.split('|') :
+    ['.mp3', '.flac', '.oga', '.wav']
+);
+
+let ambiguousTypes = new Set(
+    process.env.AMBIGUOUS_TYPES ?
+    process.env.AMBIGUOUS_TYPES.split('|') :
+     ['.webm', '.ogg']
```
Comment (itsrachelfish) [inline media-player.js]: As per my previous comment, these changes should be reverted since the media types will not change based on environment.

File: media-player.js [hunk 3]
```diff
@@ -13,16 +28,26 @@ class MediaPlayer {
...
         });
 
+
         console.log("Loaded playlist:", this.playlist);
 
         this.breakpoints = new Array(this.playlist.length);
         this.mediaLengths = new Array(this.playlist.length);
     }
 
+    isValidMediaFile(file) {
```
Comment (itsrachelfish) [inline media-player.js]: Awesome helper function! This is a great addition :+1:

File: media-player.js [hunk 4]
```diff
@@ -133,6 +158,8 @@ class MediaPlayer {
             }
         }
     }
+
```
Comment (itsrachelfish) [inline media-player.js]: Unnecessary whitespace should be removed.

File: .env [hunk 5]
```diff
@@ -0,0 +1,6 @@
+PORT=3000
```
Comment (itsrachelfish) [inline .env]: Instead of having an `.env` file tracked in git, you should create an `.env.example` file instead and add `.env` to the gitignore. This would be more consistent with our other projects and allow us to put private information in the env file that should not be tracked in git, like database passwords. https://github.com/playasoft/volunteers/blob/master/laravel/.env.example

File: .env (hunk 1 above)
Comment (AngelGzS) [reply .env]: Excelent

File: media-player.js (hunk 2 above)
Comment (AngelGzS) [reply media-player.js]: That will be reverted

File: media-player.js (hunk 3 above)
Comment (AngelGzS) [reply media-player.js]: Thank you 👍

File: .env (hunk 5 above)
Comment (AngelGzS) [reply .env]: Excelent Rachel

**Comment by AngelGzS** [comment]
All mentioned changes were done
